### ISSUE TITLE
Get mini map working

### ISSUE THREAD

**Issue by Saijin-Naib** [Saijin-Naib/UAVArena#26]

**Comment by danbjoseph** [comment]
mini map?

**Comment by Saijin-Naib** [comment]
Yeah, Leaflet Mini-Map. It is in our libs directory already. Very handy to see your current location and extent when pixel-peeping around a map so you don't get lost.

**Comment by danbjoseph** [comment]
<img width="1464" alt="Screenshot 2023-06-20 at 8 36 57 PM" src="https://github.com/Saijin-Naib/UAVArena/assets/4806884/8deda5f4-7b7f-4ce8-9625-fc5c8db45dda">

<img width="1464" alt="Screenshot 2023-06-20 at 8 37 06 PM" src="https://github.com/Saijin-Naib/UAVArena/assets/4806884/48b2185e-bb79-40cc-ae30-ccee23b11d28">

<img width="1462" alt="Screenshot 2023-06-20 at 8 37 19 PM" src="https://github.com/Saijin-Naib/UAVArena/assets/4806884/1b420dc9-5d12-4665-9528-be9f1a1ba5e3">
